examples/Darcy/darcy.jl

```diff
@@ -0,0 +1,179 @@
+
+smoothness = 1.0
+corr_length = 0.25
+dofs = 50
+
+grf = GRF.GaussianRandomField(
+    GRF.CovarianceFunction(dim, GRF.Matern(smoothness, corr_length)),
+    GRF.KarhunenLoeve(dofs),
+    pts_per_dim,
+    pts_per_dim,
+)
+
+# We define a wrapper around the GRF, and as the permeability field must be positive we introduce a domain constraint into the function distribution. Henceforth, the GRF is interfaced in the same manner as any other parameter distribution with regards to interface.
+pkg = GRFJL()
+distribution = GaussianRandomFieldInterface(grf, pkg) # our wrapper from EKP
+domain_constraint = bounded_below(0) # make κ positive
+pd = ParameterDistribution(Dict("distribution" => distribution, "name" => "kappa", "constraint" => domain_constraint)) # the fully constrained parameter distribution
+
+# Now we have a function distribution, we sample a reasonably high-probability value from this distribution as a true value (here all degrees of freedom set with `u_{\mathrm{true}} = -0.5`). We use the EKP transform function to build the corresponding instance of the ``\kappa_{\mathrm{true}}``.
+u_true = -1.5 * ones(dofs, 1) # the truth parameter
+println("True coefficients: ")
+println(u_true)
+κ_true = transform_unconstrained_to_constrained(pd, u_true) # builds and constrains the function.  
+κ_true = reshape(κ_true, N, N)
+
+# Now we generate the data sample for the truth in a perfect model setting by evaluating the the model here, and observing it by subsampling in each dimension every `obs_ΔN` points, and add some observational noise
+darcy = Setup_Param(pts_per_dim, obs_ΔN, κ_true)
+println(" Number of observation points: $(darcy.N_y)")
+h_2d = solve_Darcy_2D(darcy, κ_true)
+y_noiseless = compute_obs(darcy, h_2d)
+obs_noise_cov = 0.05^2 * I(length(y_noiseless)) * (maximum(κ_true) - minimum(κ_true))
+truth_sample = vec(y_noiseless + rand(rng, MvNormal(zeros(length(y_noiseless)), obs_noise_cov)))
+
+
+# Now we set up the Bayesian inversion algorithm. The prior we have already defined to construct our truth
+prior = pd
+
+
+# We define some algorithm parameters, here we take ensemble members larger than the dimension of the parameter space
```

Comment:
As a reader, I would like more elaboration as to why we choose ensemble members larger than the dimension of the parameter space.